builds.js

```diff
@@ -1,3 +1,7 @@
+const { clearInterval, setInterval } = require('timers');
```

Comment:
This line isn't [strictly necessary](https://nodejs.org/api/timers.html#timers_timers):

> Because the timer functions are globals, there is no need to call require('timers') to use the API.

Reply:
Ah. right. Not sure I even noticed this happening. Silly code editors doing things for me :)